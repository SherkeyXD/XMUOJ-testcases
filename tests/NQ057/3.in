37987180
20082915
47417080
393908625
156760569
550514250
32581575
120889731
389880988
912701790
68850656
229214714
433125972
25043364
171392760
32218899
16499116
441804176
664339840
83494570
67714400
127261665
497217061
103589125
150274055
535631347
243082620
222287194
62295541
18916370
87690847
216367576
221059925
397117875
397632183
224092050
113773900
153775063
879324327
10322238
489353865
61720866
109481980
500406220
297015502
757009176
438174416
8068308
281478189
307137695
529430002
147792744
288390369
296589436
93546934
350038830
187121376
250836918
43570035
183358560
349576024
88082316
358403028
133344078
540992701
57582252
224466732
86743104
118039041
9917180
539158620
661556891
566211249
62224010
121258584
139668298
140750324
193814397
48946264
54526028
38949138
21480504
344607960
659931432
766401937
54099220
66233360
626805725
502039142
101640819
599748884
13884332
136962130
3290055
333984735
236264964
181806295
16066950
38126984
503610029
102034425
272565864
48403866
446547636
30183559
192136330
83994066
764286117
406591400
170880463
355073256
28874824
607887328
418655872
372303855
40230720
119094976
571815348
441675000
67287220
370469220
646941565
69264252
569749940
430489908
