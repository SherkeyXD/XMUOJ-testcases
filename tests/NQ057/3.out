6163
4481
6886
19847
12520
23463
5708
10994
19745
30210
8297
15139
20811
5004
13091
5676
4061
21019
25774
9137
8228
11281
22298
10177
12258
23143
15591
14909
7892
4349
9364
14709
14868
19927
19940
14969
10666
12400
29653
3212
22121
7856
10463
22369
17234
27513
20932
2840
16777
17525
23009
12157
16982
17221
9671
18709
13679
15837
6600
13540
18696
9385
18931
11547
23259
7588
14982
9313
10864
3149
23219
25720
23795
7888
11011
11818
11863
13921
6996
7384
6240
4634
18563
25689
27683
7355
8138
25036
22406
10081
24489
3726
11703
1813
18275
15370
13483
4008
6174
22441
10101
16509
6957
21131
5493
13861
9164
27645
20164
13072
18843
5373
24655
20461
19295
6342
10913
23912
21016
8202
19247
25435
8322
23869
20748
